FrontEnd/src/components/ProfilePage/FormComponents/FormFields/HalfFormField.js

```diff
@@ -0,0 +1,35 @@
+import css from './HalfFormField.module.css';
+
+const HalfFormField = (props) => {
+    return (
+        <div className={css['fields__column']}>
+            <div className={css['fields__label']}>
+                {props.requredField && <label className={css['fields__label--required']}>*</label>}
+                <label
```

Comment:
Two `labels`? What for?

Reply:
I have one label. Here's one part of it just for marking the required fields with a red asterisk, and other part for the label text.